src/components/styles/Form.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: 'Montserrat';
+  src: url('https://fonts.googleapis.com/css?family=Montserrat') format('truetype');
+}
+
+.input-form {
+  display: flex;
+  flex-direction: row;
```

Comment:
- [OPTIONAL] One golden tip in styling is that we can create a class with any styling that is repeating and then give that class to any element we want. for example I can write
```
.flex {
display: flex;
}
```
and then give `flex` class to any container I want. you can apply this rule to save a lot of lines in your CSS.